Test failure possibly related?
```
FAIL: custom.enikshay.integrations.ninetyninedots.tests.test_repeaters:TestUpdatePatientPayloadGenerator.test_get_payload
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py", line 456, in test_get_payload
    payload = self._assert_payload_contains_subset(cases, expected_numbers)
  File "/mnt/commcare-hq-ro/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py", line 346, in _assert_payload_contains_subset
    self.assertDictContainsSubset(expected_payload, actual_payload)
AssertionError: Mismatched values: u'treatment_supporter_phone_number', expected: u'+9166000666', actual: u'+91066000666'
-------------------- >> begin captured logging << --------------------
debug_user_save: INFO: single,a471ffe99a85425698fd382f675e4ea5,None,/mnt/commcare-hq-ro/corehq/apps/users/models.py:96:log_user_save,/a/enikshay-bets-updates/bets/payment_confirmation
casexml.apps.phone.restore_caching: DEBUG: invalidating ('RestorePayloadPathCache', 'fake-domain-from-mixin', u'', None, u'')
casexml.apps.phone.restore_caching: DEBUG: invalidating ('RestorePayloadPathCache', 'fake-domain-from-mixin', u'', None, None)
root: DEBUG: Saving new form: XFormInstance(form_id='2055e26c974d4707b19ba739c2e78506', domain='fake-domain-from-mixin')
casexml.apps.phone.restore_caching: DEBUG: invalidating ('RestorePayloadPathCache', 'fake-domain-from-mixin', u'', None, u'')
casexml.apps.phone.restore_caching: DEBUG: invalidating ('RestorePayloadPathCache', 'fake-domain-from-mixin', u'', None, None)
root: DEBUG: Saving new form: XFormInstance(form_id='9f33839c02ce484bacf851a7a1e2318f', domain='fake-domain-from-mixin')
```

--- a/custom/enikshay/integrations/ninetyninedots/api_spec.py
+++ b/custom/enikshay/integrations/ninetyninedots/api_spec.py
@@ -241,7 +241,9 @@ def unwrap_phone_number(param, val, sector):
 
 def _parse_number(number):
     if number:
-        return phonenumbers.parse(number, "IN")
+        phone_number = phonenumbers.parse(number, "IN")
+        phone_number.italian_leading_zero = False
+        return phone_number
 
 
 def _format_number(phonenumber):

--- a/custom/enikshay/integrations/ninetyninedots/api_spec.py
+++ b/custom/enikshay/integrations/ninetyninedots/api_spec.py
@@ -250,8 +250,8 @@ def _format_number(phonenumber):
     if phonenumber:
         return phonenumbers.format_number(
             phonenumber,
-            phonenumbers.PhoneNumberFormat.INTERNATIONAL
-        ).replace(" ", "")
+            phonenumbers.PhoneNumberFormat.E164
+        )
 
 
 def noop(*args, **kwargs):

--- a/custom/enikshay/two_b_release_1/management/commands/enikshay_2b_case_properties.py
+++ b/custom/enikshay/two_b_release_1/management/commands/enikshay_2b_case_properties.py
@@ -259,9 +259,11 @@ class ENikshay2BMigrator(object):
 
         phone_number = person.get_case_property('phone_number')
         if phone_number:
+            number = phonenumbers.parse(phone_number, "IN")
+            number.italian_leading_zero = False
             props['contact_phone_number'] = strip_plus(
                 phonenumbers.format_number(
-                    phonenumbers.parse(phone_number, "IN"),
+                    number,
                     phonenumbers.PhoneNumberFormat.E164)
             )
 

